SECURITY.md

```diff
@@ -6,55 +6,63 @@ If you believe you have found a security vulnerability in any Magma Core Foundat
 
-  * Type of issue (e.g. buffer overflow, SQL injection, cross-site scripting, etc.)
-  * Step-by-step instructions to reproduce the issue
-  * Any special configuration required to reproduce the issue
-  * Impact of the issue, including how an attacker might exploit the issue
-  * Proof-of-concept or exploit code (if possible)
-  * Full paths of source file(s) related to the manifestation of the issue
-  * The location of the affected source code (tag/branch/commit or direct URL).
+- Type of issue (e.g. buffer overflow, SQL injection, cross-site scripting, etc.)
+- Step-by-step instructions to reproduce the issue
+- Any special configuration required to reproduce the issue
+- Impact of the issue, including how an attacker might exploit the issue
+- Proof-of-concept or exploit code (if possible)
+- Full paths of source file(s) related to the manifestation of the issue
+- The location of the affected source code (tag/branch/commit or direct URL).
+
+We prefer all communications to be in English.
 
 ## Response Process
 
 Certain members of the Magma team have been designated as the "vulnerability management team" and will receive any e-mails sent to `security@magmacore.org`. When receiving such an e-mail, they will:
 
-1. Reply to the e-mail acknowledging its receipt, cc'ing `security@magmacore.org` so that the other members of the team are aware that they are handling the issue.  You should receive this initial response within 72 hours, usually sooner. If for some reason you do not, please follow up via email to ensure we received your original message.
+1. Reply to the e-mail acknowledging its receipt, cc'ing `security@magmacore.org` so that the other members of the team are aware that they are handling the issue. If you are a submitter and do not receive a response within 72 hours, please follow up via email to ensure we received your original message.
 
-2. Security issues will be evaluated by the Security Vulnerability team and assigned severity level using the Common Vulnerability Scoring System as defined by [First.org](https://first.org/cvss/user-guide).  Development of remediation and fixes shall be prioritized as follows:
+2. Inform the #security channel in [the Magma Slack](https://magmacore.slack.com)
 
-   - Critical Risk: shall be treated as "P0" issue, with a fix developed and released as soon as possible even if it requires an out-or-sequence release.
-   - High Risk: shall be treated as "P1" issue, with a fix delivered no later than the next scheduled "intermediate" release.
-   - Medium Risk: shall be treated a "P1" issue, with a fix delivered no later than the next scheduled "intermediate" release.
-   - Low Risk: shall be treated as a "P2" issue, with a fix delivered no later than the next scheduled "major" release
+2. Evaluate severity using the [Scoring Rubric Template](https://docs.google.com/spreadsheets/d/1iOOgdmfHImngHE9SsOH6TV_BfzZ34C4MSKE6kaY_nRI/edit#gid=0).
```

Comment:
is the google doc owned by ospoco gdrive? i wonder if we can make this a gist or something owned by the repo?

Reply:
SECURITY.md is the one and only document

I am using a gist to check my markdown, though. You're welcome to edit:
https://gist.github.com/lucasgonze/381f26c41124e41ab4c4f0bdf8336833

Reply:
Sorry, right, you mean the rubric, not SECURITY.md. I completely agree this should be something owned by the repo.

I could make it a LibreOffice document if that is comfortable for team members. I'm personally 100% fine with that.

Reply:
What I will do is export as XSLT and commit that to magma/security.

Reply:
I presume the security classification itself is nothing which needs to be protected or non-public?
So how about putting it into the an new section within the [Github Wiki](https://github.com/magma/magma/wiki), or even in the DocuSaurus?

If there is an intend to use it as a template for reporting a security issue, why not directly creating a issue template in the respective reprository (similar to those here in the main repo)?

Reply:
> the security classification itself is nothing which needs to be protected or non-public

Correct. All the confidential data is behind the GitHub issue link.

> how about putting it into the an new section within the [Github Wiki](https://github.com/magma/magma/wiki), or even in the DocuSaurus

The worksheet is an interactive tool for measuring the severity of a vulnerability. It combines documentation on how to apply the first.org guidelines with a form for entering data about the vulnerability, then auto-generates a normalized score for the vulnerability as a whole. For example, the vulnerability in https://github.com/magma/security/issues/69 is assigned a severity score of 0.5625. This allows us to compare the severity of different vulnerabilities.

I could create a simple python script to do the calculation, though the documentation wouldn't be inline.